### PR TITLE
Add SFL button to submeta

### PR DESCRIPTION
**Pull request by OliverJAsh**

### CODE REVIEW

**Comment by OliverJAsh** [comment]
Same styles as button in top meta

image

File: common/app/views/fragments/submeta.scala.html [hunk 1]
```diff
@@ -21,8 +21,13 @@ <h2 class="submeta__head">Topics</h2>
     }
     @if(content.showBottomSocialButtons) {
         <hr/>
-        <div data-component="share">
-            @fragments.social(content)
+        <div class="u-cf">
+            <div data-component="share" class="submeta__share">
+                @fragments.social(content)
+            </div>
+            <div class="js-save-for-later submeta__save-for-later"
```
Comment (rich-nguyen) [inline common/app/views/fragments/submeta.scala.html]: `@if(ABSaveForLaterSwitch.isSwitchedOn)`

**Comment by OliverJAsh** [comment]
Ping @rich-nguyen

File: common/app/views/fragments/submeta.scala.html [hunk 2]
```diff
@@ -21,8 +22,17 @@ <h2 class="submeta__head">Topics</h2>
     }
     @if(content.showBottomSocialButtons) {
         <hr/>
-        <div data-component="share">
-            @fragments.social(content)
-        </div>
+        @if(ABSaveForLaterSwitch.isSwitchedOn) {
+            <div class="u-cf">
+        }
```
Comment (rich-nguyen) [inline common/app/views/fragments/submeta.scala.html]: `@else` ?

File: static/src/javascripts/projects/common/modules/loyalty/save-for-later.js [hunk 3]
```diff
@@ -69,22 +69,23 @@ define([
...
-            $saver = bonzo(qwery(self.classes.saveThisArticle)[0]),
-            templateName = self.templates[deferToClick ? 'signedInThisArticle' : 'signedOutThisArticle'];
-
-        $saver.html(template(templateName, {
-            url: url,
-            icon: bookmarkSvg,
-            state: state
-        }));
+    SaveForLater.prototype.renderSaveThisArticleLink = function (options) {
+        var $savers = bonzo(qwery(this.classes.saveThisArticle));
```
Comment (rich-nguyen) [inline static/src/javascripts/projects/common/modules/loyalty/save-for-later.js]: `$`

**Comment by rich-nguyen** [comment]
:+1: aside from comments

**Pull request merged**